packages/wrangler/src/queues/cli/commands/consumer/add.ts

```diff
@@ -37,6 +37,11 @@ export function options(yargs: CommonYargsArgv) {
 				type: "string",
 				describe: "Queue to send messages that failed to be consumed",
 			},
+			"max-concurrency": {
+				type: "number",
+				describe:
+					"The maximum number of concurrent consumer Worker invocations",
```

Comment:
Might be worth adding that only positive integers are valid values?

Reply:
Sure thing, added! Feel free to suggest alternative wording (anyone).